packages/react/src/components/NavBar/NavBar.tsx

```diff
@@ -9,7 +9,7 @@ import PropTypes from 'prop-types';
 import Icon from '../Icon';
 import { useId } from 'react-id-generator';
 
-interface NavBarProps {
+interface NavBarProps extends React.HTMLAttributes<HTMLDivElement> {
   children: React.ReactNode;
   initialActiveIndex?: number;
```

Comment:
I realize this wasn't part of your PR, but while you're doing some clean-up, this does not appear to be used anywhere

```suggestion
```

Reply:
I had noticed, but I guess we can include this too.